<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormWelcome</class>
 <widget class="QWidget" name="FormWelcome">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QToolButton" name="toolButton_0">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>490</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>251</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>请输入调试帐号：</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_3">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>110</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_6">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>240</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_back">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>490</y>
         <width>231</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;-</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_OK">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>110</y>
         <width>100</width>
         <height>481</height>
        </rect>
       </property>
       <property name="text">
        <string>确
定</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_9">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>370</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_1">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>110</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_4">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>240</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>321</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_2">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>110</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_7">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>370</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_8">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>370</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_5">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>240</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>:/icons/wait.jpg</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
  <include location="resouce.qrc"/>
 </resources>
 <connections/>
</ui>
